### ISSUE TITLE
Client does not work in AWS Lambda

### ISSUE THREAD

**Issue by bwindsor** [ClickSend/clicksend-python#4]
The reason for this is exactly as described in this post:
https://stackoverflow.com/questions/34005930/multiprocessing-semlock-is-not-implemented-when-running-on-aws-lambda/41541253

A solution would be for there to be an API client configuration option to turn off the use of multiprocessing pools and just make single threaded requests.

**Comment by dyllandry** [comment]
Yeah, this just derailed my project. I'll have to use another sms api. Thanks for finding this bwindsor.
